content/release-notes/2023-10-10-console.md

```diff
@@ -0,0 +1,9 @@
+### Fixes & improvements
+
+- API: Added timestamp validation for [branch creation](/docs/manage/branches#create-a-branch) to prevent selecting a time before a Neon project was created or before the defined history retention period, which is 7 days by default. The new validation also prevents selecting a time in the future. Neon supports creating point-in-time branches, but the selected time must be in the past and within the project's history retention window. For more information, see [Point-in-time restore](/docs/introduction/point-in-time-restore).
+- UI: Added a **Postgres version** column to the table on the [Projects](https://console.neon.tech/app/projects) page in the Neon console to  identify a project's Postgres version. Neon supports creating projects with Postgres versions 14, 15, or 16. Neon Pro Plan users can have multiple projects, created with any of the supported Postgres versions.
+- UI: Changed the **Explain** and **Analyze** tabs in the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor) to buttons to reflect that these features perform a query action rather than showing a different view of existing query results. For more information about these features, see [Explain and Analyze](/docs/get-started-with-neon/query-with-neon-sql-editor#explain-and-analyze).
```

Comment:
```suggestion
- UI: Improved the **Explain** and **Analyze** tabs in the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor) to buttons to better reflect that these features perform query actions to get new results, rather than just show a different view of existing query results. For more information about these features, see [Explain and Analyze](/docs/get-started-with-neon/query-with-neon-sql-editor#explain-and-analyze).
```

Reply:
I'll keep "Changed" here since we changed the tabs to buttons. Merged the other suggestion. Thank you!